app/scripts/modules/ecs/src/serverGroup/configure/wizard/capacityProvider/CapacityProvider.tsx

```diff
@@ -61,6 +89,24 @@ class EcsCapacityProvider extends React.Component<IEcsCapacityProviderProps, IEc
     this.setState({ capacityProviderStrategy: capacityProviderStrategy });
   };
 
+  private updateCapacityProviderStrategy = (targetCapacityProviderType: string) => {
+    const useDefaultCapacityProviders = targetCapacityProviderType == 'defaultCapacityProvider';
```

Comment:
Is it possible to pass in a boolean here? I'm hesitant doing string comparison (on the off chance that type changes in the future, or the case changes, etc). If you can pass in true/false since that's really all you care about, that would improve maintainability.

Reply:
Resolved.